microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md

```diff
@@ -59,38 +61,40 @@ To distribute your extension using the Windows registry:
    > [!NOTE]
    > If you want to install an extension from the Chrome Web Store, set the value of `update_url` to `https://clients2.google.com/service/update2/crx`.
 
-1. Verify that your extension is listed in Microsoft Edge, by going to `edge://extensions`.
+1. In Microsoft Edge, go to `edge://extensions`, and then verify that your extension is listed.
 
 
 <!-- ====================================================================== -->
-## Use a preferences JSON file (macOS and Linux)
+## Using a preferences JSON file (macOS and Linux)
+
+To distribute your extension by using a preferences JSON file:
 
-To distribute your extension using a preferences JSON file:
+1. When using Linux, make sure your `.crx` extension file is available on the machine that the extension will be installed on.  Copy the `.crx` extension file to a local directory, or use a network share that is reachable from the machine.
 
-1. When using Linux, ensure your `.crx` extension file is available on the machine that the extension will be installed on. Copy the `.crx` extension file to a local directory, or use a  network share that is reachable from the machine.
+1. Create a JSON file where the name of the file corresponds to the ID of your extension. For example, create a JSON file with the file name `aaaaaaaaaabbbbbbbbbbcccccccccc.json`.<!--30 letters, ok?-->
```

Comment:
Engineering, 

The aaa...ccc filename shown is only 30 chars, not 32.  Add two chars to the example string?  Or change "32" to "30" in the following sentence?

Reply:
We should make it 32 characters by adding 2 chars to the example string

Reply:
In 3 spots in this .md, changed to aaaaaaaabbbbbbbbccccccccdddddddd  (8 chars x 4 = 32 chars)